### ISSUE TITLE
Display the pools a post is in on the post page

### ISSUE THREAD

**Issue by CounterClops** [rr-/szurubooru#692]
I'm not super familiar with frontend dev, so I couldn't find how to make a list of linked pools be displayed on a individual post page easily. I tried looking, but I don't know where the start.

I can see that the list of pools is provided by the backend when the client calls `https://<domain>/api/post/<id>` with a `pools` attribute, so the client should have the information when rendering the page.  Something like a collapsible **Pools** section either above or below **Tags** that only shows if the post has any linked pools.

It's a feature that makes it easier to find out if a post I'm looking at is apart of a pool.

**Comment by Deer-Spangle** [comment]
Is this something improved or fixed by #650?

**Comment by CounterClops** [comment]
> Is this something improved or fixed by #650?

Yeah that would be great, as it looks to do exactly what I would be aiming for. Should've gone through the PRs before raising this, though that PR looks like it hasn't been touched in ages.
